examples-watchdog/CMakeLists.txt

```
@@ -3,9 +3,9 @@ add_executable(patch_eeprom patch_eeprom.c)
 target_link_libraries(patch_eeprom ftdi1)
 
```

Suggestion: Use `CMAKE_INSTALL_*` variables. For this we need to include `GNUInstallDirs`

```suggestion

include(GNUInstallDirs)
```